ext/reflection/php_reflection.c

```diff
@@ -4181,10 +4185,8 @@ ZEND_METHOD(ReflectionClass, getAttributes)
 
 	GET_REFLECTION_OBJECT_PTR(ce);
 
-	if (ce->type == ZEND_USER_CLASS && ce->info.user.attributes) {
-		attributes = ce->info.user.attributes;
-		scope = ce;
-	}
+	attributes = ce->attributes;
+	scope = ce;
```

Comment:
I think the additional aliasing here is not needes, just pass `ce->attributes` and `ce` to `reflect_attributes()` instead.